charts/invenio/templates/_helpers.tpl

```diff
@@ -5,6 +5,24 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
```

Comment:
```suggestion
If release name contains chart name, the release name will be used as a full name.
```

Reply:
I just copy paste the output of `helm create`, didn't even read it 😂